tools/passwordGenerator.php

```diff
@@ -10,9 +12,28 @@ function generatePassword() {
     $("#password")[0].value = password;
 }
 
+
+async function copyToClipboard(){
```

Comment:
![image](https://github.com/Relluem94s/RelluToolbox/assets/6181042/3f2317e2-ff07-43ba-ae33-ece52bba33ac)
![image](https://github.com/Relluem94s/RelluToolbox/assets/6181042/37a72465-0e81-4256-b1f9-a5b58031b6b3)

do you get the same error? i'm using firefox@fedora & chrome@fedora

Reply:
You need to grant Permissions for Clipboard in Chrome, like for Camera or Microphone Usage